src/components/MapAndTableControls/MapAndTableControls.jsx

```diff
@@ -36,10 +37,35 @@ const ZoomToSecondaryButton = styled(ButtonSecondary)`
   align-items: center;
 `
 
+const CloseButton = styled(MapAndTableControlsButtonSecondary)`
+  width: 100%;
+  justify-content: center;
+  & svg {
+    height: 25px;
+    width: 25px;
+  }
+`
+
+const FilterControlContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  & > div:nth-child(2) {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
Can this be a separate styled component? If it can, should it be?

Reply:
Maybe? Im rusty with styled components. I suppose its more explicit for troubleshooting, but where do you draw that line? Busting one-off things up also makes things disjointed?

Do you have more than mild feelings on it? If so, I can do it no prob. If this is just for discussion, I have no opinion. Is avoiding child selection in favour of a named styled thing a convention/best practice? @julia-rom what are your thoughts on where this line is?

Reply:
I found an existing `Row` component that does the job since its simple, but still curious about how you guys feel about where the line is :)